stripe_payments.php

```diff
@@ -663,6 +685,8 @@ public function authorizeStoredCc(
             'capture_method' => 'manual',
             'setup_future_usage' => 'off_session'
         ];
+        if($customer_present)
+            $payment['setup_future_usage'] = 'on_session';
```

Comment:
So https://stripe.com/docs/api/payment_intents/create#create_payment_intent-setup_future_usage seems to indicate that this would attach a new payment method to a user but not authorize it for use unless the customer is present.  Am I reading that wrong?